judge/handler.py

```diff
@@ -106,29 +115,31 @@ def process_problem(code: str, contest: int, name: str, statement: str, input_fo
             code=code, contest=c, name=name, statement=statement,
```

Comment:
One last try: "To create and save an object in a single step, use the create() method." Save would fail here as compilation_script and test_script can be None.

Reply:
Ahhhhhh, got it. Reverting this change. Sorry.

Reply:
Reverted the change